build.sh

```diff
@@ -1,25 +1,18 @@
-#!/bin/sh
+#!/bin/bash
+set -eo pipefail
 
-echo "update submodule"
+echo "update submodules of PV"
 git submodule update --init --recursive
 
 echo "compile bpftool and copy it"
 make -C bpftool/src
 cp bpftool/src/bpftool /usr/sbin
```

Comment:
Move it to build.rs like libxdp

Reply:
I got it. I moved these lines into build.rs to run the same commands while building application